<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>380</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Download SVG</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>440</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Thickness</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>380</y>
      <width>61</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Shape:</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>380</y>
      <width>131</width>
      <height>33</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Rectangle</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Polygon</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>380</y>
      <width>64</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>Units:</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_2">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>400</y>
      <width>81</width>
      <height>33</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Inches</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Centimetres</string>
     </property>
    </item>
   </widget>
   <widget class="QComboBox" name="comboBox_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>470</y>
      <width>81</width>
      <height>33</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>.25 inch</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>.125 inch</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>460</y>
      <width>64</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>Length:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>500</y>
      <width>64</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>Height:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>540</y>
      <width>64</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>Width:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>430</y>
      <width>64</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>Color:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="colorText">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>420</y>
      <width>131</width>
      <height>33</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lengthText">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>450</y>
      <width>131</width>
      <height>33</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="heightText">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>500</y>
      <width>131</width>
      <height>33</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="widthText">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>540</y>
      <width>131</width>
      <height>33</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>781</width>
      <height>361</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_2"/>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
